cli_reference/openshift_cli/usage-oc-kubectl.adoc

```diff
@@ -21,6 +21,41 @@ The `oc` binary offers a built-in `login` command that allows authentication and
 +
 The additional command `oc new-app`, for example, makes it easier to get new applications started using existing source code or pre-built images. Similarly, the additional command `oc new-project` makes it easier to start a project that you can switch to as your default.
 
+If you installed an earlier version of the `oc` binary, you cannot use it to complete all of the commands in {product-title} {product-version}.
+
+[IMPORTANT]
+====
+You must download and install the latest version of the `oc` binary corresponding to your {product-title} server version.
+====
+
+Non-security API changes will involve, at minimum, two minor releases (4.1 to 4.2 to 4.3, for example) to allow older `oc` binaries to update. Using new capabilities may require newer `oc` binaries. A 4.3 server may have additional
+capabilities that a 4.2 `oc` binary cannot use and a 4.3 `oc` binary may have additional capabilities that are unsupported by a 4.2 server.
+
+.Compatibility Matrix
+
+[cols="1,1,1"]
+|===
+
+|
+|*X.Y* (`oc` binary)
+|*X.Y+N* footnote:versionpolicyn[Where *N* is a number greater than 1.] (`oc` binary)
```

Comment:
I see we've had "greater than 1" since [way back in 4.1][1], but shouldn't this be `>= 1`?  Because as the text above points out:

> A 4.3 server may have additional capabilities that a 4.2 `oc` binary cannot use and a 4.3 `oc` binary may have additional capabilities that are unsupported by a 4.2 server.

And in that case, `N` is 1.

[1]: https://github.com/openshift/openshift-docs/commit/3a3ebbb9a87777330910df047193fd14b1c4d36f#diff-efca4885ed001b9a0a7182b68bfd9b7f6a06236201e9ec3db3be46bbd6861310R36